README.md

```diff
@@ -106,18 +91,16 @@ sh
 
```

Comment:
## Your Sections :red_circle:
![image](https://user-images.githubusercontent.com/93039359/219051754-603ae0f7-4fd2-4bf8-8217-d4a0d4366077.png)

## Descriptive Sections :green_circle:
![image](https://user-images.githubusercontent.com/93039359/219052339-3f3a9275-c973-4395-8e1a-9d3840c92114.png)


- [ ] These two sections **Show your support**, and **Acknowledgement** are empty :fearful: Kindly fill these sections to make your readme file descriptive and professional :blush: